src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/PhysicalAddress.cs

```diff
@@ -21,17 +20,16 @@ public PhysicalAddress(byte[] address)
 
         public override int GetHashCode()
         {
-            if (_hashNotComputed)
+            if (_hash != 0)
             {
-                _hashNotComputed = false;
-                _hash = 0;
+                var hash = 0;
```

Comment:
```suggestion
                int hash = 0;
```

Reply:
Why not `var` ?

Reply:
Our guidelines for the repo are that var is only allowed when the type is named on the right-hand side, namely a new expression or an explicit cast. vars otherwise make code much harder for us to read and review.

Reply:
Done